src/server/migrations/20201129124100_create_groups_table.js

```diff
@@ -0,0 +1,20 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('groups', (table) => {
+    table.increments();
+    table.string('name').notNullable();
+    table
+      .integer('user_id')
+      .unsigned()
+      .notNull()
+      .references('id')
+      .inTable('users')
+      .onDelete('cascade');
```

Comment:
Remove this property. It's the other way around.
Need to add `group_id` to `users` instead.